Dockerfile

```diff
@@ -0,0 +1,34 @@
+## golang image created 2021-06-24T00:31:06.02014601Z 
+FROM docker.io/library/golang@sha256:be99fa59acd78bb22a41bbc1e15ebfab2262498ee0c2e28c3d09bc44d51d1774 AS builder
+WORKDIR /go/src/github.com/komish/preflight
+COPY . .
+RUN make build
+
+# ubi-minimal image created 2021-06-01T12:12:46.922866Z
+FROM registry.access.redhat.com/ubi8/ubi-minimal@sha256:0ccb9988abbc72d383258d58a7f519a10b637d472f28fbca6eb5fab79ba82a6b
+
+# Add preflight binary
+COPY --from=builder /go/src/github.com/komish/preflight/preflight /usr/local/bin/preflight
+
+# Install dependencies
+RUN microdnf install \
+    buildah \
+    bzip2 \
+    gzip \
+    iptables \
+    podman \
+    python3-pip \
+    skopeo
+
+# Install OpenShift client binary
+RUN curl -L https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux-4.7.18.tar.gz | tar -xzv -C /usr/local/bin --exclude=README.md --exclude=kubectl
```

Comment:
can we ensure to remove the tar file after unzipping?

Reply:
The curl call is being piped to tar, so as far as I know the file should not be there after the operation completes. Do you see it in the resulting image somewhere?

Reply:
i think since its being piped no intermediate tar file gets saved..

Reply:
Great! Sorry my OCD kicks in whenever I see `curl` or `wget`